benchmarks/src/main/java/org/apache/druid/benchmark/query/SqlBenchmark.java

```diff
@@ -111,16 +114,20 @@ public void setup()
         .createQueryRunnerFactoryConglomerate();
     final QueryRunnerFactoryConglomerate conglomerate = conglomerateCloserPair.lhs;
     final PlannerConfig plannerConfig = new PlannerConfig();
+    final DruidLeaderClient druidLeaderClient = EasyMock.createMock(DruidLeaderClient.class);
 
     this.walker = new SpecificSegmentsQuerySegmentWalker(conglomerate).add(dataSegment, index);
     plannerFactory = new PlannerFactory(
         CalciteTests.createMockSchema(conglomerate, walker, plannerConfig),
+        new TestServerInventoryView(walker.getSegments()),
```

Comment:
hm `TestServerInventoryView` is already in `/druid/sql/src/main/java/org/apache/druid/sql/calcite/util`. And not sure if I should rename it in this PR, as it was already existing.

Reply:
i didn't look carefully, it's in `/druid/sql/src/test/java/org/apache/druid/sql/calcite/util`, I think we can keep it there, as it's mostly being used by tests. Will use a mock in this benchmark instead of `TestServerInventoryView`.